data/en/qcas_0018.json

```diff
@@ -41,8 +41,8 @@
             "id": "qcas-section",
             "title": "QCAS",
             "groups": [{
-                "id": "introduction-group",
-                "title": "Introduction",
+                "id": "qcas-group",
```

Comment:
maybe just `qcas` as this shows in the url?

Reply:
I think its also worth not putting `-block` in any of the block ids for all surveys. Makes URL more meaningful. Something to consider for future.